backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/spreadsheet/process/ExportToSpreadsheetProcess.java

```diff
@@ -108,7 +112,15 @@ private void updateDatevAcctExport()
 		{
 			final int recordId = getRecord_ID();
 			final I_DatevAcctExport datevAcctExport = InterfaceWrapperHelper.create(getCtx(), recordId, I_DatevAcctExport.class, getTrxName());
-			datevAcctExport.setExportDate(SystemTime.asTimestamp());
+
+			final AdProcessId adProcessId = getProcessInfo().getAdProcessId();
+			final I_AD_Process processRecord = adProcessDAO.getById(adProcessId);
+			final String processValue = processRecord != null ? processRecord.getValue() : "<NULL>";
+
+			if (processValue.equals("AccountingExportBodymed_ExportLinesToExcel"))
```

Comment:
please create a `ProcessId` constant and use that, rather than `AD_Process.Value`. This value might be edited in the apllication dictionary

Reply:
or even better, introduce a process parameter called `IsUpdateExportDate`.

In the standard process you can define it with default value `N` and DisplayLogic `1=0` (hide it).

In your new process you can define it with default value `Y` and DisplayLogic `1=0` (hide it).

@metas-ts @riadmen  wdyt?

Reply:
note: if you already developed Tobi's suggestion, i am fine with that too.